CRM/Member/Import/Parser/Membership.php

```diff
@@ -666,6 +666,17 @@ public function membership_format_params($params, &$values, $create = FALSE) {
             }
           }
         }
+        if ($type == 'Autocomplete-Select') {
```

Comment:
@sushantpaste @eileenmcnaughton 

This looks like repetitive code and could be centralised in one place without having to do fixes in all components separately.
https://github.com/civicrm/civicrm-core/blob/master/CRM/Event/Import/Parser/Participant.php#L527
https://github.com/civicrm/civicrm-core/blob/master/CRM/Contribute/Import/Parser/Contribution.php#L666

so that all components correctly format custom data for all field types.

Reply:
@yashodha I totally agree!